src/server/handlers/openapi/mojaloop/consents/{ID}.ts

```diff
@@ -20,19 +20,39 @@
 
  * Google
  - Steven Wijaya <stevenwjy@google.com>
+ - Abhimanyu Kapur <abhi.kapur09@gmail.com>
  --------------
  ******/
 
 import { Request, ResponseToolkit } from '@hapi/hapi'
 import { Handler, Context } from 'openapi-backend'
 import { logger } from '~/shared/logger'
+import { consentRepository } from '~/repositories/consent'
 
-export const put: Handler = async (context: Context, request: Request, h: ResponseToolkit) => {
+export const put: Handler = async (
+  context: Context,
+  request: Request,
+  h: ResponseToolkit
+) => {
   logger.logRequest(context, request, h)
+  // Updates consent fields
+  consentRepository.updateConsentById(
+    context.request.params.ID as string,
```

Comment:
there is inconsistency of handling context.request.params.ID, sometimes there is inline casting, whereas in other cases const is declared (with casting) and used as function argument. Let keep consistency across all files